manual/cops_style.md

```diff
@@ -611,10 +611,11 @@ This cop checks for uses of class variables. Offenses
 are signaled only on assignment to class variables to
 reduce the number of offenses that would be reported.
 
-Setting value for class variable need to take care.
-If some class has been inherited by other classes, setting value
-for class variable affected children classes.
-So using class instance variable is better in almost case.
+You have to be careful when setting a value for a class
+variable; if a class has been inherited, changing the 
```

Comment:
This line contains  an trailing space, I'm not sure why this line contains it. Can you retry `bundle exec rake generate_cops_documentation`? In my environment the trailing space is removed by the rake task.

Reply:
@koenpunt Ping :-)